surrogates/youtube-iframe-api.js

```diff
@@ -45,7 +50,18 @@
         const { height, width, videoId, playerVars = { }, events } = config;
 
         if (!(target instanceof Element)) {
-            target = document.getElementById(target);
+            const orignalTarget = target;
+            target = document.getElementById(orignalTarget);
+
+            if (!target) {
+                for (const videoElement of allVideoElements()) {
+                    // eslint-disable-next-line eqeqeq
```

Comment:
We want the type conversion in this instance, but I think generally this is a good linting rule. So I've added an exception for this line.